templates/etc/nginx/sites-available/site-available.j2

```diff
@@ -19,6 +19,14 @@ server {
 }
 
 server {
+    gzip on;
+    gzip_disable "msie6";
+    gzip_vary on;
+    gzip_comp_level 6;
+    gzip_http_version 1.1;
+    gzip_min_length 256;
+    gzip_types text/plain text/css application/json application/x-javascript text/xml application/xml application/xml+rss text/javascript application/vnd.ms-fontobject application/x-font-ttf font/opentype image/svg+xml image/x-icon;
+
```

Comment:
The whole block is situated in the big `ssl` `if`. If needs to be moved down to line 52 https://github.com/pyslackers/ansible-role-nginx/pull/20/files#diff-91bdf8b1c20d6e8c771744d6e40ce195R52

Reply:
Good catch, I didn't realize how you set that up. That's a very cool way to do it!